project3/index.html

```diff
@@ -0,0 +1,409 @@
+                 style="stroke-width: 2px; fill: rgb(250, 250, 250);" />
+         </svg>
+         <header>SUPER HERO <br>REGISTERY </header>
+         <h1>With great power there must also come- <br>great responsibility</h1>
+         <div>
+
+             <form action="">
+
+
+
+                 <h2>
+                     <legend>IDENTITY</legend>
+                 </h2>
+
+                 <label for="real-name">Real Name</label>
+
+                 <input id="real-name" type="text" placeholder="" required>
+
+
+                 <label for="tel-number">Telephone Number</label>
+
+                 <input id="tel-number" type="number" placeholder="" required>
+
+
+                 <label for="email">Email</label>
+
+                 <input id="email" type="email" placeholder="" required>
+
+
+                 <h2>
+                     <legend>PERSONAL CHARACTERISTICS</legend>
+                 </h2>
+                 <label for="age">Age</label>
+                 <br>
+                 <select name="" id="age">
+                     <option value="18">18</option>
+                     <option value="19">19</option>
+                 </select>
+
+                 <fieldset>
+                     <legend>Category</legend>
+
+                     <input id="alien" type="checkbox">
+                     <label for="alien">Alien</label>
+                     <br>
+
+                     <input id="cyborg" type="checkbox">
+                     <label for="cyborg">Cyborg</label>
+                     <br>
+
+                     <input id="experiment" type="checkbox">
+                     <label for="experiment">Experiment</label>
+                     <br>
+
+                     <input id="human" type="checkbox">
+                     <label for="human">Human</label>
+                     <br>
+
+                     <input id="inhuman" type="checkbox">
+                     <label for="inhuman">Inhuman</label>
+                     <br>
+
+                     <input id="mutant" type="checkbox">
+                     <label for="mutant">Mutant</label>
+                     <br>
+                 </fieldset>
+
+                 <fieldset>
+                     <legend>Gender</legend>
+
+                     <input name="gender" id="woman" type="radio">
```

Comment:
As per the [requirements](https://mica-web.github.io/gd-220/proj3.html#requirements), this question is required. You can add the `required` attribute to one or all of the `input`s with the shared `name="gender"`, like so:
```
<input name="gender" id="woman" type="radio" required>
```
just like you did on line 42.